Core/include/Acts/EventData/TrackState.hpp

```diff
@@ -8,7 +8,6 @@
 
 #pragma once
 
-#include "Acts/EventData/Measurement.hpp"
```

Comment:
Why do you remove the include here?

Reply:
I forgot that this was still in the branch. Initially I wanted to see where and how we use measurements and to get a better overview I removed all non-mandatory includes of it. In this case, the `MeasurementHelper` provides it to this class. I can put them back in if you want it.

Reply:
Yeah, please remove the change. (include what you need)

Reply:
Done